include/Dialect/Pulse/Utils/Utils.h

```diff
@@ -37,6 +37,16 @@ namespace mlir::pulse {
 Waveform_CreateOp getWaveformOp(PlayOp pulsePlayOp,
                                 CallSequenceOp callSequenceOp);
 
+Waveform_CreateOp
+getWaveformOp(PlayOp pulsePlayOp,
+              std::deque<mlir::pulse::CallSequenceOp> &callSequenceOpStack);
```

Comment:
We have number of usages of `std::deque<mlir::pulse::CallSequenceOp>` now for the CallSequence stacks. It might be useful to define a `CallSequenceStack_t` in order to help with readability. Something like:

Reply:
Hmm drop the suggestion:

```
using CallSequenceStack_t = std::deque<mlir::pulse::CallSequenceOp>;
```

and then 

```
Waveform_CreateOp getWaveformOp(PlayOp pulsePlayOp, CallSequenceStack_t &callSequenceOpStack);
```